echo/server/dembrane/conversation_health.py

```
@@ -20,50 +20,74 @@
-    logger.debug(f"Project language is {project_language}")
 
+def _should_skip_diarization(project_language: str) -> bool:
+    """
+    Determines whether diarization should be skipped for a given project language.
+    
+    Returns True if diarization is disabled for non-English languages based on configuration; otherwise, returns False.
+    """
     if DISABLE_MULTILINGUAL_DIARIZATION and project_language != "en":
-        logger.debug(f"Skipping diarization for chunk {chunk_id} because project language is {project_language}")
-        return None
+        logger.debug(f"Skipping diarization because project language is {project_language}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)